python/lsst/sims/maf/batches/srdBatch.py

```diff
@@ -109,12 +114,17 @@ def astrometryBatch(colmap=None, runName='opsim',
                    'order': 0, 'caption': None}
     # Expected error on parallax at 10 AU.
     for rmag in (20.0, 24.0):
+        if rmag == 20.0:
```

Comment:
For something more compact could we do:
~~~
for rmag, plotmax in zip((20,24),(2,15)):
~~~

Also, should `plotmax` just be `xMax` so it is more descriptive? I see that it is also used to set the `colorMax` so perhaps `plotmax` is better.

Reply:
Good points! Left it as plotmax though, because yeah, it's max for color and x. but could be convinced otherwise.